components/banner/index.js

```diff
@@ -0,0 +1,60 @@
+import Router from 'next/router'
+import TokenSaleDetails from './tokenSaleDetails';
+import Countdown from './countdown';
+import stylesheet from './banner.scss'
+import { getSecondsUntilNextPeriod } from '../constants'
+
+class Banner extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      active: this.props.currentDayServer ? 2 : 1,
```

Comment:
I don't know what this means 😕

If you are using numbers to represent specific values, please use a `const` definition to describe what these numbers mean.

e.g.
```
const DRAWER_IS_ON_SCREEN = 2;
const DRAWER_IS_OFF_SCREEN = 1
const isDrawerActive = isActive() ? DRAWER_IS_ON_SCREEN : DRAWER_IS_OFF_SCREEN
```